src/test/frontend/deploy/createnamespace_controller_test.js

```diff
@@ -26,14 +26,6 @@ describe('Deploy controller', () => {
     });
   });
 
-  it('should return true if namespace is empty', () => {
```

Comment:
Hmm... Can you test namespacePattern regex? I know this is a constant, but with regexes I'm always worried that they are wrong. Just assert a few positive and negative cases.